examples/src/main/scala/com/vesoft/nebula/connector/ConnectorReaderExample.scala

```diff
@@ -42,11 +42,14 @@ object ConnectorReaderExample {
     val vertexDataset: Dataset[Row] =
       spark.read
         .nebula("127.0.0.1:45500", "nb", "100")
```

Comment:
host and port should be setting

Reply:
nebula's parameters are required, other parameters not required can be set using withXX.